src/main/drivers/compass_hmc5883l.h

```diff
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include "drivers/io.h"
```

Comment:
It's just for consistency. Some driver headers have the directory prefix, some don't. I don't have a preference for either, but it is good for them to all be the same.